ova/Libraries/provision-opendistro.sh

```diff
@@ -0,0 +1,293 @@
+searchguard_version="1.8"
+resources_url=https://raw.githubusercontent.com/wazuh/wazuh-documentation/${BRANCH}
+
+logger() {
+
+    echo $1
+}
+
+startService() {
+
+    if [ -n "$(ps -e | egrep ^\ *1\ .*systemd$)" ]; then
+        systemctl daemon-reload 
+        systemctl enable $1.service 
+        systemctl start $1.service 
+        if [ "$?" != 0 ]; then
+            logger "${1^} could not be started."
+            exit 1;
+        else
+            logger "${1^} started"
+        fi
+    elif [ -n "$(ps -e | egrep ^\ *1\ .*init$)" ]; then
+        chkconfig $1 on 
+        service $1 start 
+        /etc/init.d/$1 start 
+        if [ "$?" != 0 ]; then
+            logger "${1^} could not be started."
+            exit 1;
+        else
+            logger "${1^} started"
+        fi
+    elif [ -x /etc/rc.d/init.d/$1 ] ; then
+        /etc/rc.d/init.d/$1 start 
+        if [ "$?" != 0 ]; then
+            logger "${1^} could not be started."
+            exit 1;
+        else
+            logger "${1^} started"
+        fi
+    else
+        logger "Error: ${1^} could not start. No service manager found on the system."
+        exit 1;
+    fi
+}
+
+## Show script usage
+getHelp() {
+
+   echo ""
+   echo "Usage: $0 arguments"
+   echo -e "\t-d   | --debug Shows the complete installation output"
+   echo -e "\t-i   | --ignore-health-check Ignores the health-check"
+   echo -e "\t-h   | --help Shows help"
+   exit $1 # Exit script after printing help
+}
+
+
+## Install the required packages for the installation
+installPrerequisites() {
+    logger "Installing all necessary utilities for the installation..."
+    $sys_type install curl unzip wget libcap -y -q 
+    echo -e '[AdoptOpenJDK] \nname=AdoptOpenJDK \nbaseurl=http://adoptopenjdk.jfrog.io/adoptopenjdk/rpm/centos/$releasever/$basearch\nenabled=1\ngpgcheck=1\ngpgkey=https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public' | tee /etc/yum.repos.d/adoptopenjdk.repo 
+    $sys_type install adoptopenjdk-11-hotspot -y -q 
+    export JAVA_HOME=/usr/
+
+    if [ "$?" != 0 ]; then
+        logger "Error: Prerequisites could not be installed"
+        exit 1;
+    else
+        logger "Done"
+    fi
+}
+
+## Add the Wazuh repository
+addWazuhrepo() {
+    WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+    logger "Adding the Wazuh repository..."
+    rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH 
+    if [ "${STATUS_PACKAGES}" = "prod" ]; then
+        logger "Adding production repository..."
+        rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH
+        echo -e "[wazuh_repo]\ngpgcheck=1\ngpgkey=https://packages.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages.wazuh.com/${WAZUH_MAJOR}.x/yum/\nprotect=1" | tee /etc/yum.repos.d/wazuh.repo 
+    elif [ "${STATUS_PACKAGES}" = "dev" ]; then
+        logger "Adding development repository..."
+        rpm --import https://packages-dev.wazuh.com/key/GPG-KEY-WAZUH 
```

Comment:
```suggestion
    rpm --import https://packages.wazuh.com/key/GPG-KEY-WAZUH 
    if [ "${STATUS_PACKAGES}" = "prod" ]; then
        logger "Adding production repository..."
        echo -e "[wazuh_repo]\ngpgcheck=1\ngpgkey=https://packages.wazuh.com/key/GPG-KEY-WAZUH\nenabled=1\nname=EL-$releasever - Wazuh\nbaseurl=https://packages.wazuh.com/${WAZUH_MAJOR}.x/yum/\nprotect=1" | tee /etc/yum.repos.d/wazuh.repo 
    elif [ "${STATUS_PACKAGES}" = "dev" ]; then
        logger "Adding development repository..."
```

Reply:
Done in 599737656bcaaa119a2c30c76c1760fa0590643a